bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1071,6 +1071,13 @@ Toggle for mexico city specific logic.
 Max number of plant functional types in naturally vegetated landunit.
 </entry>
 
+<entry id="n_dom_pfts" type="integer" category="physics"
+       group="clm_inparm"
+       valid_values="0,1,2,3,4,5,6,7,8,9,10,11,12,13,14" value="0">
```

Comment:
Add note that 0, means "do nothing". It essentially means the same as setting it to "15", which you aren't allowed to set it to.

Reply:
Done!